packages/terra-image/src/variants/_DecorativeImage.jsx

```diff
@@ -0,0 +1,198 @@
+  DEFAULT: 'default',
+  ROUNDED: 'rounded',
+  CIRCLE: 'circle',
+  THUMBNAIL: 'thumbnail',
+};
+
+const FitTypes = {
+  COVER: 'cover',
+  SCALEDOWN: 'scale-down',
+  FILL: 'fill',
+  CONTAIN: 'contain',
+  NONE: 'none',
+};
+
+const propTypes = {
+  /**
+   * The source string for the image which will be loaded and displayed.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * Sets the style of the image from the following values; `default`, `rounded`, `circle`, `thumbnail`.
+   */
+  variant: PropTypes.oneOf(['default', 'rounded', 'circle', 'thumbnail']),
+  /**
+   * Sets the fluid behavior of the image, which is `nonfluid` by default.
+   */
+  isFluid: PropTypes.bool,
+  /**
+   * A React element which will be displayed during loading and in case of src load failure.
+   */
+  placeholder: PropTypes.element,
+  /**
+   * Sets the height of the image.
+   */
+  height: PropTypes.string,
+  /**
+   * Sets the width of the image.
+   */
+  width: PropTypes.string,
+  /**
+   * Function to be executed when the image load is successful.
+   */
+  onLoad: PropTypes.func,
+  /**
+   * Function to be executed when the image load errors.
+   */
+  onError: PropTypes.func,
+  /**
+  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
+  */
+  fit: PropTypes.oneOf(['cover', 'scale-down', 'fill', 'contain', 'none']),
+};
+
+/* eslint-disable react/default-props-match-prop-types */
+const defaultProps = {
+  variant: 'default',
+  isFluid: false,
+  fit: 'fill',
+};
+/* eslint-enable react/default-props-match-prop-types */
+
+class DecorativeImage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.ImageRef = React.createRef();
+
+    this.state = {
+      isLoading: true,
+      isError: false,
+      prevPropsSrc: props.src,
+    };
+
+    this.handleOnLoad = this.handleOnLoad.bind(this);
+    this.handleOnError = this.handleOnError.bind(this);
+  }
+
+  componentDidUpdate() {
+    objectFitImages(this.ImageRef.current);
+  }
+
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.src !== prevState.prevPropsSrc) {
+      return {
+        isLoading: true,
+        isError: false,
+        prevPropsSrc: nextProps.src,
+      };
+    }
+
+    return null;
+  }
+
+  handleOnLoad() {
+    this.setState({ isLoading: false });
+    const { onLoad } = this.props;
+
+    if (onLoad !== undefined) {
+      onLoad();
+    }
+  }
+
+  handleOnError() {
+    this.setState({ isLoading: false, isError: true });
+    const { onError } = this.props;
+
+    if (onError !== undefined) {
+      onError();
+    }
+  }
+
+  createImage(customProps, imageClasses) {
+    const {
+      src, height, width,
+    } = this.props;
+
+    const additionalParams = customProps;
+    // removes Aria attributes and title from customProps
+    Object.keys(additionalParams).forEach(prop => {
+      if (prop.includes('aria') || prop === 'title') {
```

Comment:
What about the 'alt' attribute? I think that should also be prevented.

Reply:
```suggestion
      if (prop.includes('aria') ||  ['title', 'alt'].includes(prop)) {
```

Reply:
[updated](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-11aa68ac49c8e6adecf93efdff5cee647a3363465db473c721bf1afe3cf1e4ab)